apps/libc/dynamic/qsort.c

```diff
@@ -301,29 +301,29 @@ Return Value:
         //
 
         CompareResult =
-                      CompareFunction(ArrayBase + (SmallerIndex * ElementSize),
+                      CompareFunction((char *)ArrayBase + (SmallerIndex * ElementSize),
                                       LastElement);
 
         if (CompareResult == 0) {
             SmallestIndex += 1;
-            QUICKSORT_SWAP(ArrayBase, ElementSize, SmallestIndex, SmallerIndex);
+            QUICKSORT_SWAP((char *)ArrayBase, ElementSize, SmallestIndex, SmallerIndex);
         }
 
         CompareResult =
                       CompareFunction(LastElement,
-                                      ArrayBase + (LargerIndex * ElementSize));
+								(char *)ArrayBase + (LargerIndex * ElementSize));
```

Comment:
What happened to the spacing here? I can't see from the Github UI if these are tabs, but if they are, please don't use tabs. We are a "spaces only" project :)